platforms/documentation/docs/src/docs/userguide/optimizing-builds/build-cache/build_cache.adoc

```diff
@@ -280,6 +280,8 @@ The remote build cache can be configured by specifying the type of build cache t
 
 The built-in local build cache, link:{groovyDslPath}/org.gradle.caching.local.DirectoryBuildCache.html[DirectoryBuildCache], uses a directory to store build cache artifacts.
 By default, this directory resides in the Gradle User Home, but its location is configurable.
+It is also possible to disable the build cache entirely,
+or to disabling storing of the build outputs there.
```

Comment:
❓ See other comment, also "to disabling storing" should be corrected
And ideally this short sentence can be on a single line.